acra-censor/acra-censor_test.go

```diff
@@ -384,6 +384,32 @@ func testWhitelistValuePattern(t *testing.T) {
 			t.Fatal(err, "Blacklist pattern passed query. \nPattern: ", pattern, "\nQuery: ", query)
 		}
 	}
+
+	whitelist.Reset()
+	pattern = "SELECT * from t where ID = %%VALUE%%"
+	err = whitelist.AddPatterns([]string{pattern})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	acceptableQueries = []string{
+		"SELECT a, b FROM t WHERE ID = 'someValue_testValue_1234567890'",
+		"SELECT a, b FROM t WHERE ID = 'someValue'",
```

Comment:
better to test different type of values instead same with different length/value. can we check boolean/integer/float/nullable values too? `where a=1 or b=1.0 or c=TRUE or d=NULL

Reply:
plus we should check that it works with >1 conditions - `where a=1 and b=2`, not only with one.